.cbmc-batch/jobs/aws_cryptosdk_md_abort/README.md

```diff
@@ -0,0 +1,3 @@
+# Memory safety proof for aws_cryptosdk_md_abort
+
+This proof harness attains 100% code coverage.
```

Comment:
If the proof has 100% coverage, you can remove the README file. There is nothing to justify on coverage.

Reply:
Okay, yeah I thought it might be good for documentation but this info is on the spreadsheet so the README adds nothing extra